internal/cmd/kafka/command_acl_create_onprem.go

```diff
@@ -19,25 +19,25 @@ func (c *aclCommand) newCreateCommandOnPrem() *cobra.Command {
 		Example: examples.BuildExampleString(
 			examples.Example{
 				Text: "You can specify only one of the following flags per command invocation: `--cluster-scope`, `--consumer-group`, `--topic`, or `--transactional-id`. For example, for a consumer to read a topic, you need to grant \"read\" and \"describe\" both on the `--consumer-group` and the `--topic` resources, issuing two separate commands:",
-				Code: "confluent kafka acl create --allow --principal User:Jane --operation read --operation describe --consumer-group java_example_group_1",
+				Code: "confluent kafka acl create --allow --principal User:Jane --operations read --operations describe --consumer-group java_example_group_1",
```

Comment:
It should actually be `--operation` and not `--operations`, see https://docs.confluent.io/confluent-cli/current/command-reference/kafka/acl/confluent_kafka_acl_create.html#flags.

However, this example is a bit concerning since it looks like the `--operation` flag can't be passed more than once (its type is "string", not "strings")... Let's fix that in this PR!

(We could probably change this example to only grant the "read" operation)

Reply:
Thanks @brianstrauch I checked here: https://docs.confluent.io/confluent-cli/current/command-reference/kafka/acl/confluent_kafka_acl_create.html#flags. To further clear up the customer's confusion mentioned in this Docs Jira: https://confluentinc.atlassian.net/browse/DOCS-22308, so _--operations_ is used for Cloud and _--operation_ is used for on-prem? Looks like the customer was trying to correlate the release note (ttps://docs.confluent.io/confluent-cli/current/release-notes.html#confluent-cli-v3-0-0-release-notes) with what's depicted in the examples in our docs.

Reply:
And cool, I'll make the change you mentioned above

Reply:
Yes, in Cloud you're able to grant more than one operation at once but on-prem only lets you grant one at a time.

Reply:
Got it. I have responded to the customer.